scripts/add-javascript-rules.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
```

Comment:
That's ok I am not used to work with Git Bash but this is working (the fix for jq.exe is nice :))